source/server/options_impl.cc

```diff
@@ -11,6 +11,10 @@
 #include "spdlog/spdlog.h"
 #include "tclap/CmdLine.h"
 
+#if defined(__APPLE__)
+#define uint64_t unsigned long
```

Comment:
`uint64_t` exists but is typedef `unsigned long long`. Without this you get this error:

```
In file included from source/server/options_impl.cc:12:
In file included from external/envoy_deps/thirdparty/tclap/include/tclap/CmdLine.h:27:
In file included from external/envoy_deps/thirdparty/tclap/include/tclap/SwitchArg.h:30:
In file included from external/envoy_deps/thirdparty/tclap/include/tclap/Arg.h:54:
external/envoy_deps/thirdparty/tclap/include/tclap/ArgTraits.h:80:22: error: type 'unsigned long long' cannot be used prior to '::' because it has no members
    typedef typename T::ValueCategory ValueCategory;
                     ^
external/envoy_deps/thirdparty/tclap/include/tclap/ValueArg.h:403:37: note: in instantiation of template class 'TCLAP::ArgTraits<unsigned long long>' requested here
        ExtractValue(_value, val, typename ArgTraits<T>::ValueCategory());
                                           ^
external/envoy_deps/thirdparty/tclap/include/tclap/ValueArg.h:363:5: note: in instantiation of member function 'TCLAP::ValueArg<unsigned long long>::_extractValue' requested here
                                _extractValue( args[*i] );
                                ^
source/server/options_impl.cc:31:29: note: in instantiation of member function 'TCLAP::ValueArg<unsigned long long>::processArg' requested here
  TCLAP::ValueArg<uint64_t> base_id(
                            ^
1 error generated.
```

Swapping out the typedef I added for `using std::uint64_t;` produces an identical error.